src/interpret.cpp

```diff
@@ -574,10 +574,11 @@ unsigned invokeNativeSlow(Thread* t, GcMethod* method, void* function)
   uint64_t result;
 
   if (DebugRun) {
+    signed char *cname = method->class_() && method->class_()->name() ? method->class_()->name()->body().begin() : (signed char*) "?";
```

Comment:
I guess since `method->class_()->name()->body().begin()` returns an `int8_t*`, you'll still need a reinterpret_cast in there, but at least you should be able to avoid a const_cast by adding `const` to the declared type.